### PR TITLE
LibJS: Add FIXMEs to a few functions that need UTF-16 handling

### PR DESCRIPTION
**Pull request by nico**

### CODE REVIEW

File: Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 1]
```diff
@@ -319,6 +321,7 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::substring)
     if (interpreter.argument_count() == 0)
         return js_string(interpreter, string);
 
+    // FIXME: index_start and index_end should index a UFF-16 codepoint view of the string.
```
Comment (awesomekling) [inline Libraries/LibJS/Runtime/StringPrototype.cpp:324]: Typo, UFF-16

File: Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 2]
```diff
@@ -381,6 +385,7 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::slice)
...
```
Comment (awesomekling) [inline Libraries/LibJS/Runtime/StringPrototype.cpp:388]: Typo, UFF-16

File: Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 3]
```diff
@@ -429,10 +434,10 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::last_index_of)
         return {};
     if (search_string.length() > string.length())
         return Value(-1);
-
     auto max_index = string.length() - search_string.length();
     auto from_index = max_index;
     if (interpreter.argument_count() >= 2) {
+        // FIXME: from_index should index a UFF-16 codepoint view of the string.
```
Comment (awesomekling) [inline Libraries/LibJS/Runtime/StringPrototype.cpp:440]: Also here

File: Libraries/LibJS/Runtime/StringPrototype.cpp (hunk 1 above)
Comment (nico) [reply Libraries/LibJS/Runtime/StringPrototype.cpp:324]: It's German: https://en.wiktionary.org/wiki/uff ...and apparently Norwegian: https://en.wikipedia.org/wiki/Uff_da

**Pull request merged**